.travis.yml

```diff
@@ -42,7 +42,7 @@ script:
   - yarn test-viewer
   - yarn test-lantern
   - yarn i18n:checks
-  # _JAVA_OPTIONS is breaking parsing of compiler output. See #3338.
+  # _JAVA_OPTIONS is breaking parsing of `compile-devtools` output. See #3338.
```

Comment:
changed this because I assumed this was old leftover closure stuff and originally deleted it, but #3338 is about a failure on `yarn compile-devtools` (which may eventually run again depending on the FIXME below)